docs/guide.md

```diff
@@ -417,20 +417,20 @@ sudo passwd {user} # replace user with your code-server user
 
 [![Cloudflared](https://img.shields.io/badge/Cloudflared-E4863B?style=for-the-badge&logo=cloudflare&logoColor=ffffff)](https://github.com/cloudflare/cloudflared)
 
-1.  Install [cloudflared](https://github.com/cloudflare/cloudflared#installing-cloudflared) on your local computer
-2.  Then go to `~/.ssh/config` and add the following:
+1.  Install [cloudflared](https://github.com/cloudflare/cloudflared#installing-cloudflared) on your local computer and remote server
+2.  Then go to `~/.ssh/config` and add the following on your local computer:
 
 ```shell
 Host *.trycloudflare.com
 HostName %h
-User root
```

Comment:
Hmm...could this have unintended consequences? @code-asher

Reply:
I actually prefer to use a non-root user login in the local configuration file for security purposes. Also the username entered in vscode should be the same as the user in the file.

Reply:
Good point, this should be the user that is running code-server and not root.  Looks good to me.  :+1: